common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -243,10 +245,46 @@ class ModuleI18nService(object):
     i18n service.
 
     """
+    def __init__(self, block=None):
+        """
+        We are using this block for things like locating the block's translation domain (PO+MO files). Here we attempts
+         to look up the provided string in the XBlock's own domain translation catalog, currently expected to be:
+            <xblock_root>/conf/locale/<language>/LC_MESSAGES/<domain>.po
+        If ModuleI18nService can't locate the domain translation catalog then we fall-back onto
+        django.utils.translation.ugettext, which will attempt to find a matching string in the
```

Comment:
we can remove the "ugettext" specific context from this docstring, because it's now applicable across multiple operations